packages/antify-admin/src/server/datasources/auth/schema.prisma

```diff
@@ -61,11 +61,11 @@ model RolePermission {
 }
 
 model UserTenantAccess {
-  user      User     @relation(fields: [userId], references: [id])
+  user      User     @relation(fields: [userId], references: [id], onDelete: Cascade)
   userId    String
-  tenant    Tenant   @relation(fields: [tenantId], references: [id])
+  tenant    Tenant   @relation(fields: [tenantId], references: [id], onDelete: Cascade)
   tenantId  String
-  role      Role     @relation(fields: [roleId], references: [id])
+  role      Role     @relation(fields: [roleId], references: [id], onDelete: Cascade)
```

Comment:
When a role get deleted, the users access to the tenant should not get deleted.

Solution 1: Do not allow to delete a role when one or more tenantAccesses exists.
Solution 2: On deleting a role, all associated tenantAccesses get replaced with a default role

Reply:
Fixed and added toaster if role is still assigned to a user.